.github/workflows/tests.yml

```diff
@@ -56,6 +56,14 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox v5.0.1 installation
```

Comment:
i suggest to install the bleeding edge (master) version so we don’t need to update that link every time SCT updates

Reply:
I agree, that's what I originally intended to do. However, the name of the zip file containing the latest version of SCT is changed each time, and so is its URL. I don't know how to call it in the same way as dcm2niix, with /releases/latest/download/name.zip for example

Reply:
you can simply use git, which is installed in the travis runner:
~~~
git clone https://github.com/neuropoly/spinalcordtoolbox
cd spinalcordtoolbox
yes | ./install_sct
~~~

Reply:
Much more efficient! I'll try that, thanks